cli/src/pcluster/validators/utils.py

```diff
@@ -12,7 +12,54 @@
 # This module contains all the classes representing the Resources objects.
 # These objects are obtained from the configuration file through a conversion based on the Schema classes.
 #
+from typing import Any, Union
+
+BOOLEAN_VALUES = ["true", "false"]
 
 
 def get_bucket_name_from_s3_url(import_path):
     return import_path.split("/")[2]
+
+
+def str_to_bool(string: str = None) -> bool:
+    """Convert string to boolean value.
+
+    Args:
+        string: String to convert. Defaults to None.
+
+    Returns:
+        True if string is "true" (case-insensitive), False otherwise.
+    """
+    return str(string).lower() == "true"
+
+
+def is_boolean_string(value: str) -> bool:
+    """Check if value is a valid boolean string.
+
+    Args:
+        value: String value to check.
+
+    Returns:
+        True if value is "true" or "false" (case-insensitive), False otherwise.
+    """
+    return str(value).lower() in BOOLEAN_VALUES
```

Comment:
[Minor]Just seems redundant to keep converting a string when its already a string